src/dmd/hdrgen.d

```diff
@@ -984,7 +984,13 @@ public:
     {
         buf.writestring("align ");
         if (d.ealign)
-            buf.printf("(%s) ", d.ealign.toChars());
+        {
+            buf.printf("(%s)", d.ealign.toChars());
+            AttribDeclaration ad = cast(AttribDeclaration)d;
+            if (ad.decl && ad.decl.dim == 1)
```

Comment:
I think that this is going to wrongly elide the space for empty `align` declarations:

```d
align (1) {}
```

Reply:
Changed. I also fixed duplicated code introduced by #1139 .